<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressContent</class>
 <widget class="QWidget" name="ProgressContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="dailyBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>dailies</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="wedWeeklyBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string>weeklies</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="monWeeklyBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string>monday weeklies</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
